cli/command/image/build.go

```diff
@@ -643,3 +650,49 @@ func imageBuildOptions(dockerCli command.Cli, options buildOptions) types.ImageB
 		Platform:       options.platform,
 	}
 }
+
+func parseOutputs(inp []string) ([]types.ImageBuildOutput, error) {
+	var outs []types.ImageBuildOutput
+	if len(inp) == 0 {
+		return nil, nil
+	}
+	for _, s := range inp {
+		csvReader := csv.NewReader(strings.NewReader(s))
+		fields, err := csvReader.Read()
+		if err != nil {
+			return nil, err
+		}
+		if len(fields) == 1 && fields[0] == s {
+			outs = append(outs, types.ImageBuildOutput{
+				Type: "local",
+				Attrs: map[string]string{
+					"dest": s,
+				},
+			})
+			continue
+		}
+
+		out := types.ImageBuildOutput{
+			Attrs: map[string]string{},
+		}
+		for _, field := range fields {
+			parts := strings.SplitN(field, "=", 2)
+			if len(parts) != 2 {
+				return nil, errors.Errorf("invalid value %s", field)
```

Comment:
We don't have (or will have) options that don't require a value? Otherwise we may defer this to the daemon, and use 
https://github.com/docker/cli/blob/9e712073274021ef1e552ecbb54595142b49edbb/opts/parse.go#L40-L53

Reply:
The API already accepts these values in a structured form and correctly formatted. We also need to parse out the values that are handled on the client side.